Packs/Microsoft365Defender/IncidentFields/incidenttype-Microsoft_365_Defender_Incident.json

```diff
@@ -0,0 +1,27 @@
+{
+    "autorun": false,
+    "color": "#F8C89A",
+    "days": 0,
+    "daysR": 0,
+    "default": false,
+    "detached": false,
+    "disabled": false,
+    "extractSettings": {
+        "fieldCliNameToExtractSettings": {},
+        "mode": "Specific"
+    },
+    "hours": 0,
+    "hoursR": 0,
+    "id": "Microsoft 365 Defender Incident",
+    "layout": "b91a81ab-4cb6-4df2-8a24-696eeb9a7633",
```

Comment:
not sure what is the layout here - if this is the layout you added let's change this to its id and why do other fields not have this?

Reply:
This file is auto generated I have no why the layout field appears only in this file. Removed the field, working great.